### PR TITLE
automl: add base model samples for automl ga

### PR DESCRIPTION
**Pull request by nnegrey**

### CODE REVIEW

File: automl/cloud-client/deploy_model_test.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    if version_info.minor == 5:
+        MODEL_ID = "TEN5112482778553778176"
+    elif version_info.minor == 6:
+        MODEL_ID = "TCN3472481026502981088"
+    elif version_info.minor == 7:
+        MODEL_ID = "TST8532792392862639819"
+elif version_info.major == 2:
+    MODEL_ID = "TEN1499896588007374848"
+else:
+    MODEL_ID = "TEN7450981283112419328"
```
Comment (nnegrey) [inline automl/cloud-client/deploy_model_test.py]: Note: Experimentation to get tests pass when running 4 python versions in parallel.

File: automl/cloud-client/delete_model.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    # TODO(developer): Uncomment and set the following variables
+    # project_id = 'YOUR_PROJECT_ID'
+    # model_id = 'YOUR_MODEL_ID'
+
+    client = automl.AutoMlClient()
+    # Get the full path of the model.
+    model_full_id = client.model_path(project_id, "us-central1", model_id)
+    response = client.delete_model(model_full_id)
+
+    print("Model deleted. {}".format(response.result()))
```
Comment (gguuss) [inline automl/cloud-client/delete_model.py]: (Optional) As per the Authoring guide, I prefer single quotes over double.

File: automl/cloud-client/deploy_model.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    # [START automl_deploy_model]
+    from google.cloud import automl
+
+    # TODO(developer): Uncomment and set the following variables
+    # project_id = 'YOUR_PROJECT_ID'
+    # model_id = 'YOUR_MODEL_ID'
+
+    client = automl.AutoMlClient()
+    # Get the full path of the model.
+    model_full_id = client.model_path(project_id, "us-central1", model_id)
```
Comment (gguuss) [inline automl/cloud-client/deploy_model.py]: The double quotes look a bit odd.

File: automl/cloud-client/deploy_model_test.py [hunk 4]
```diff
@@ -0,0 +1,35 @@
+# Copyright 2018 Google LLC
```
Comment (gguuss) [inline automl/cloud-client/deploy_model_test.py]: 2019 copyright on new files, if this is an update, 2018 is fine

File: automl/cloud-client/get_model_evaluation.py [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+    # [START automl_language_entity_extraction_get_model_evaluation]
+    # [START automl_language_sentiment_analysis_get_model_evaluation]
+    # [START automl_language_text_classification_get_model_evaluation]
+    # [START automl_translate_get_model_evaluation]
+    # [START automl_vision_classification_get_model_evaluation]
+    # [START automl_vision_object_detection_get_model_evaluation]
+    from google.cloud import automl
+
+    # TODO(developer): Uncomment and set the following variables
+    # project_id = 'YOUR_PROJECT_ID'
```
Comment (gguuss) [inline automl/cloud-client/get_model_evaluation.py]: If you're going to change to double quotes elsewhere, you should also change these here.

File: automl/cloud-client/get_model_evaluation.py [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+    client = automl.AutoMlClient()
+    # Get the full path of the model evaluation.
+    model_evaluation_full_id = client.model_evaluation_path(
+        project_id, "us-central1", model_id, model_evaluation_id
+    )
+
+    # Get complete detail of the model evaluation.
+    response = client.get_model_evaluation(model_evaluation_full_id)
+
+    print("Model evaluation name: {}".format(response.name))
```
Comment (gguuss) [inline automl/cloud-client/get_model_evaluation.py]: Something about the quotes looks wrong.

File: automl/cloud-client/get_model_evaluation.py [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+    print("\tnanos: {}".format(response.create_time.nanos / 1e9))
+    print(
+        "Evaluation example count: {}".format(response.evaluated_example_count)
+    )
+    # [END automl_language_sentiment_analysis_get_model_evaluation]
+    # [END automl_language_text_classification_get_model_evaluation]
+    # [END automl_translate_get_model_evaluation]
+    # [END automl_vision_classification_get_model_evaluation]
+    # [END automl_vision_object_detection_get_model_evaluation]
+    print(
```
Comment (gguuss) [inline automl/cloud-client/get_model_evaluation.py]: This style of wrapping looks odd.

I prefer the following:
```
    print('Entity extraction model evaluation metrics: {}'.format(
        response.text_extraction_evaluation_metrics))
```

File: automl/cloud-client/get_model_test.py [hunk 8]
```diff
@@ -0,0 +1,26 @@
+# Copyright 2018 Google LLC
```
Comment (gguuss) [inline automl/cloud-client/get_model_test.py]: FYI copyright date

File: automl/cloud-client/get_model_evaluation_test.py [hunk 9]
```diff
@@ -0,0 +1,46 @@
+# Copyright 2018 Google LLC
```
Comment (gguuss) [inline automl/cloud-client/get_model_evaluation_test.py]: FYI copyright date

File: automl/cloud-client/list_model_evaluations.py [hunk 10]
```diff
@@ -0,0 +1,94 @@
...
+    # model_id = 'YOUR_MODEL_ID'
+
+    client = automl.AutoMlClient()
+    # Get the full path of the model.
+    model_full_id = client.model_path(project_id, "us-central1", model_id)
+
+    print("List of model evaluations:")
+    for evaluation in client.list_model_evaluations(model_full_id, ""):
+        print("Model evaluation name: {}".format(evaluation.name))
+        print(
```
Comment (gguuss) [inline automl/cloud-client/list_model_evaluations.py]: Weird print style blocks

File: automl/cloud-client/list_model_evaluations.py [hunk 11]
```diff
@@ -0,0 +1,94 @@
...
+            "Model annotation spec id: {}".format(
+                evaluation.annotation_spec_id
+            )
+        )
+        print("Create Time:")
+        print("\tseconds: {}".format(evaluation.create_time.seconds))
+        print("\tnanos: {}".format(evaluation.create_time.nanos / 1e9))
+        print(
+            "Evaluation example count: {}".format(
```
Comment (gguuss) [inline automl/cloud-client/list_model_evaluations.py]: Weird double quotes

File: automl/cloud-client/list_model_evaluations_test.py [hunk 12]
```diff
@@ -0,0 +1,27 @@
+# Copyright 2018 Google LLC
```
Comment (gguuss) [inline automl/cloud-client/list_model_evaluations_test.py]: double check Copyright year

File: automl/cloud-client/list_models.py [hunk 13]
```diff
@@ -0,0 +1,47 @@
...
+# limitations under the License.
+
+
+def list_models(project_id):
+    """List models."""
+    # [START automl_list_models]
+    from google.cloud import automl
+
+    # TODO(developer): Uncomment and set the following variables
+    # project_id = 'YOUR_PROJECT_ID'
```
Comment (gguuss) [inline automl/cloud-client/list_models.py]: If you're adopting double-quotes, use them consistently.

File: automl/cloud-client/undeploy_model_test.py [hunk 14]
```diff
@@ -0,0 +1,35 @@
+# Copyright 2018 Google LLC
```
Comment (gguuss) [inline automl/cloud-client/undeploy_model_test.py]: Copyright year.

**Comment by nnegrey** [comment]
Yea, I usually run `black *.py -l 79`
Yea, I'm not a fan of the two nits you pointed out. 

Mostly I just started using black after I noticed the generated libraries use it, so I figured I'd conform.

**Comment by nnegrey** [comment]
Happy to fix the black formatting if we aren't adopting it in our style.

**Comment by gguuss** [comment]
> Mostly I just started using black after I noticed the generated libraries use it, so I figured I'd conform.
Meh, there are a lot of people on the team who are in favor of using Black despite all of it's stylistic shortcomings, just because it is easy to run. I'm ok with this style until we have an alternative for auto-formatting. I may look into customizing out the Black features that are triggering my style preferences in particular:

* Double-quotes
* The extra newlines at the start / end of ( ... ) blocks
* Column wrapping at 88
* The extra ',' characters added to the last members of function definitions on continuations
* Newlines added to closing )'s

And any others that I find. Ideally, the auto-formatter we adopt should be configurable to our styles.

**Pull request merged**